src/uipath/_cli/cli_run.py

```diff
@@ -108,6 +109,7 @@ async def execute():
     type=click.Path(exists=True),
     help="File path for the .json input",
 )
+@track
```

Comment:
We should exclude runtime runs (eg env having a job id)

Reply:
See list of full envs from a run:

https://uipath.atlassian.net/wiki/spaces/Agents/pages/88737415226/Environment+Variables